data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,773 @@
 
-event "hr: media retraction"
+mission "Hai Reveal Exploration Period: Concerned Hai"
+	minor
+	landing
+	invisible
+	source
+		government "Hai"
+		not attributes "uninhabited" "retirement"
+	to offer
+		has "Hai Reveal: Meet the Team: Sayari 2: done"
+		not "event: hr: exploration period"
+		random < 20
+	on offer
+		conversation
+			`Upon arrival, your way is briefly blocked by a loud commotion with a small crowd of people. Two Hai, a shopkeeper and a merchant captain, are arguing loudly with each other outside the spaceport convenience outlet. You can't tell what they're saying - but the store, a chain whose name has been translated to "Spacorns" for the benefit of human visitors, has a big sign up in both languages that reads: "ONE Haycorn Snus pack per customer. No Exceptions. ID Required."`
+			`	It appears to be almost concluded, and a moment later the captain makes a disgusted noise, taps his pad, swipes his solitary pack, and stalks off into the spaceport looking altogether a lot like a frustrated predator.`
+			choice
+				`	(Ask someone what's going on.)`
+					goto "query an onlooker"
+				`	(Talk to the shopkeeper.)`
+			`	Stepping into the store, the Hai has only just returned to behind the counter where he's carefully putting back another pack of the Haycorn Snus. Now getting a closer look at him, his fur is mussed, and it seems that he might have had to wrestle the second pack away from the captain. He's certainly on edge, for he jumps with a little start as he hears you enter.`
+			`	"Ah, Captain, sorry about all that. What can I do for you?"`
+			choice
+				`	"Did you have to fight that man?"`
+				`	"What's with the sign?"`
+					goto "shop sign"
+			`	He takes a deep breath to steady himself before responding. "Unfortunately, yes. Fortunately for me, he was one of those captains who forget that walking up and down the length of your ship doesn't really qualify as regular exercise.`
+			label "shop sign"
+			`	"It's all a very unfortunate situation at the moment. With the increased appearance of humans in our space, and the uncertainty over what's going on, some merchants aren't plying the trade they usually do back and forth. The supplies of a few select things that are imported from the Republic have not been appearing quite as often, and so some Hai have taken to stockpiling certain commodities in case the supply shuts off entirely." He looks pointedly at the snus packs. "Tobacco is one of those commodities. Regulations prevent it from being grown on any Hai worlds. So, unless someone goes to the effort of setting up a greenhouse in space, that's entirely imported."`
+			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`
```

Comment:
"Math" is indeed the US version of "maths."

Adding "but" really isn't appropriate with how it changes the meaning here, though, because "not quite the same effect as humans" is an explanation of the math boosting, not a contradiction to it.

Reply:
Ah, I see. How about just the maths change then?
```suggestion
			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for math by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`
```